db/migrate/20160413140153_create_topic_section_guides.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopicSectionGuides < ActiveRecord::Migration
+  def change
+    create_table :topic_section_guides do |t|
+      t.integer :topic_section_id, null: false
+      t.integer :guide_id, null: false
```

Comment:
Indexing these two columns would be good. They'll be used a lot because of the has_many :through.